python/dgl/distributed/dist_dataloader.py

```diff
@@ -327,9 +327,11 @@ def add_edge_attribute_to_graph(g, data_name):
             The graph.
         data_name : str
             The name of data that's stored in DistGraph.ndata/edata.
+        gb_padding : int, optional
+            The padding value for GraphBolt partitions' new edge_attributes.
```

Comment:
Can you add a doc string about what does the default value `0` mean for gb_padding?

Reply:
And what does edge_attributes mean and how it is used?